### PR TITLE
[Merged by Bors] - fix: dont treat as trained if no intents (PL-1040) [bugfix]

### PR DESCRIPTION
**Pull request by trs**
Don't consider the version trained if it does not contain any intents.

### CODE REVIEW

File: lib/services/classification/predictor.class.ts [hunk 1]
```diff
@@ -362,6 +360,13 @@ export class Predictor extends EventEmitter {
 
       this.predictions.result = 'llm';
 
+      if (!(llmPrediction.predictedIntent in this.intentNameMap)) {
```
Comment (theProf) [inline lib/services/classification/predictor.class.ts:363]: is this case not caught [here](https://github.com/voiceflow/general-runtime/blob/fecd85cf2cecac4de55acd445615df659bebfd84/lib/services/classification/predictor.class.ts#L216) in the llm func itself?

**Review by theProf — APPROVED** [review]

File: lib/services/classification/predictor.class.ts (hunk 1 above)
Comment (trs) [reply lib/services/classification/predictor.class.ts:363]: I had this occur during testing, so I added this check 🤔

Comment (theProf) [reply lib/services/classification/predictor.class.ts:363]: ah true, my link if pre-prompt. but maybe this logic can go with the [post-checks](https://github.com/voiceflow/general-runtime/blob/fecd85cf2cecac4de55acd445615df659bebfd84/lib/services/classification/predictor.class.ts#L268) in llm?

**Comment by trs** [comment]
bors merge

**Pull request closed without merging**